test/functional/ui/inccommand_spec.lua

```diff
@@ -1354,6 +1354,23 @@ describe("inccommand=nosplit", function()
       :echo 'foo'^         |
     ]])
   end)
+
+  it("only executes the first command in preview", function()
+    feed(':%s/two/three/g|q!')
+    screen:expect([[
+      Inc substitution on |
+      {12:three} lines         |
+      Inc substitution on |
+      {12:three} lines         |
+                          |
+      {15:~                   }|
+      {15:~                   }|
+      {15:~                   }|
+      {15:~                   }|
+      :%s/two/three/g|q!^  |
+    ]])
+    eq(eval('v:null'), eval('v:exiting')) 
```

Comment:
`make testlint` found trailing whitespace on this line, that's why the lint build is failing.

You can always check `make lint` to run all linters locally.

Reply:
May be I missed it, I usually do this